cmd/2019_7/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/afarbos/aoc/pkg/test"
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+func TestFindMaxThrusterSignal04(t *testing.T) {
+	testData := map[int][]int{
+		43210: {3, 15, 3, 16, 1002, 16, 10, 16, 1, 16, 15, 15, 4, 15, 99, 0, 0},
+		54321: {3, 23, 3, 24, 1002, 24, 10, 24, 1002, 23, -1, 23, 101, 5, 23, 23, 1, 24, 23, 23, 4, 23, 99, 0, 0},
+		65210: {3, 31, 3, 32, 1002, 32, 10, 32, 1001, 31, -2, 31, 1007, 31, 0, 33, 1002, 33, 7, 33, 1, 33, 31,
+			31, 1, 32, 31, 31, 4, 31, 99, 0, 0, 0},
+	}
+	for resExpected, instructions := range testData {
+		test.EqualInt(t, findMaxThrusterSignal04(instructions), resExpected)
+	}
+}
+
+func TestFindMaxThrusterSignal59(t *testing.T) {
+	testData := map[int][]int{
+		139629729: {3, 26, 1001, 26, -4, 26, 3, 27, 1002, 27, 2, 27, 1, 27, 26, 27, 4, 27, 1001, 28, -1, 28,
+			1005, 28, 6, 99, 0, 0, 5},
+		18216: {3, 52, 1001, 52, -5, 52, 3, 53, 1, 52, 56, 54, 1007, 54, 5, 55, 1005, 55, 26, 1001, 54, -5, 54,
+			1105, 1, 12, 1, 53, 54, 53, 1008, 54, 0, 55, 1001, 55, 1, 55, 2, 53, 55, 53, 4, 53, 1001, 56,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 105 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)